<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightUI</class>
 <widget class="QDialog" name="LightUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>172</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>3</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LightUI</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>402</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="outputFileSelectionLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="holovibes::gui::Drag_drop_lineedit" name="OutputFilePathLineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select output file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="OutputFileBrowseToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Start/Stop</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionStart_Stop">
   <property name="text">
    <string>Start_Stop</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop the recording</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>holovibes::gui::Drag_drop_lineedit</class>
   <extends>QLineEdit</extends>
   <header>gui_drag_drop_lineedit.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionStart_Stop</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OutputFileBrowseToolButton</sender>
   <signal>clicked()</signal>
   <receiver>LightUI</receiver>
   <slot>browse_record_output_file</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_record_output_file</slot>
  <slot>update_record_file_path</slot>
  <slot>start_stop_recording</slot>
 </slots>
</ui>
